web/src/components/context/NRFPreferencesContext.tsx

```diff
@@ -58,7 +54,8 @@ export function NRFPreferencesProvider({
 }: {
   children: React.ReactNode;
 }) {
-  const [theme, setTheme] = useLocalStorageState<string>(
+  const { setTheme: setNextThemesTheme } = useTheme();
+  const [theme, setThemeState] = useLocalStorageState<string>(
```

Comment:
need to import from https://github.com/onyx-dot-app/onyx/blob/f473b85acdcac2a1384631829fbfdad212204353/web/src/lib/types.ts#L15
```suggestion
  const [theme, setThemeState] = useLocalStorageState<ThemePreference>(
```